### PR TITLE
WIP Writting MFractor Microsoft Blog post

### PR DESCRIPTION
**Pull request by ravero**
Hi @matthewrdev,
I've created this Pull Request so you can check the work in progress on the writing of the blog post.

### CODE REVIEW

**Comment by matthewrdev** [comment]
@ravero This is a great foundation for the blog; writing a marketing piece is very, very challenging. 

Here is my feedback and requests:
 
 * Please remove the initial section that focuses on myself. The strategic goal of the blog is to emphasise the Windows product and any part that doesn't focus on that is not strictly necessary.
 * The blog should showcase only the Windows extension. Therefore, all screenshots need to be of the Windows product, not of the Mac product. Can you please replace all Mac screenshots with their Windows equivalent?
 * Please remove any feature reference that is not in the Windows product. For example, image asset completion is a Mac only feature for the moment, so we can't advertise it for the Windows extension. An alternate feature would be Image Tooltips or C# color tooltips.
 * For all screenshots, please use the default text theme. Most developers use the out-of-the-box text colouring and we need our screenshots to align with their Visual Studio experience. This makes the product instantly visible and removes the potential distraction of a developer going "Well, it looks like it changes the text colouring, maybe it's not for me". Basically, everything needs to be as plain as possible to reduce any miscommunication.
 * For all GIFs, please reduce the feature usage speed by about half or replace with static images. The current GIFs are too quick and thus are hard to follow.

Can you please take screenshots on Windows of the following features:

 * Extract style tool.
 * Matching style detection code issue tooltip.
 * Missing data-binding tooltip. (https://raw.githubusercontent.com/mfractor/mfractor.content/master/mfractor-for-visual-studios-release/img/data-binding-analysis.png)
 * Data-binding intellisense. (https://raw.githubusercontent.com/mfractor/mfractor.content/master/mfractor-for-visual-studios-release/img/data-binding-intellisense.png)
 * Insert/delete grid row/column refactoring.
 * Grid row/column index analysis.

If you make the changes requested, I'll merge the PR and apply some editing.

**Comment by ravero** [comment]
There's a bug happening on Windows that hadn't allowed me to take a screenshot of the image tooltip. In this example you can see that it identifies `mfractor_logo` as an error but the tooltip show a fix with the same name, and tooltip preview is never shown:

Screen Shot 2020-07-20 at 16 11 28

**Comment by ravero** [comment]
I have replaced most of the images with the Windows counterpart. Missing only gifs and code references.

**Comment by ravero** [comment]
I've replaced all the current images with the Windows counterparts.
Working on the other stuff.

**Comment by matthewrdev** [comment]
@ravero This is shaping up very nicely.

Review notes:

 * Change `Xamarin.Forms is a beloved by thousands of developers around the globe by its powerful cross-platform UI capabilities` to `Xamarin.Forms is beloved by developers around the globe for its powerful cross-platform capabilities`. This shortens the sentence and doesn't suggest any number of developers who use Xamarin.Forms. Pretty sure that James/Xamarin team will flag that as well.
* For the `MFractor uses common conventions to identify where each of these parts are.` sentence, note that MFractor understand industry standard naming practices to automatically discover view models for Views. Please also link to our discovery rules. (https://docs.mfractor.com/xamarin-forms/binding-context-resolution/overview/).
 * When mentioning a feature, link to the relevant documentation if it exists.
 * Move the Ready For MVVM section to be the first major highlight. We will focus on these benefits first.

I'll add additional review notes this evening.

**Comment by ravero** [comment]
@matthewrdev please take a look at changes on the commit.
